src/app/component/eco-news/components/news-list/news-list.component.ts

```diff
@@ -72,9 +75,21 @@ export class NewsListComponent implements OnInit, OnDestroy {
   private addElemsToCurrentList(): void {
     if (this.tagsList) {
       this.ecoNewsSubscription = this.ecoNewsService.getNewsListByTags(this.currentPage, this.numberOfNews, this.tagsList)
+        .pipe(
+          catchError((error) => {
+            this.snackBar.openSnackBar('Oops, something went wrong. Please reload page or try again later.', 'X', 'red-snackbar');
+            return error;
```

Comment:
Please, leave an ampty line before the 'return'

Reply:
fixed